src/Tools/dotnet-trace/CommandLine/Commands/CollectCommand.cs

```diff
@@ -113,7 +113,7 @@ private static async Task<int> Collect(CancellationToken ct, IConsole console, i
                     }
                     if (File.Exists(output.FullName))
                     {
-                        Console.Error.WriteLine("Unable to create file.");
+                        Console.Error.WriteLine($"Unable to create file {output.FullName}. Try using -o or --output option to specify a different output file name.");
```

Comment:
TBH, I'm not sure we want to error out on file exists. What do other profilers/tracing tools do? Of the top of my head, they attempt to overwrite, but FWIW, I do believe the current error message is not great neither.

Reply:
Hmm that is a good point. PerfCollect just overwrites the existing file. And  PerfView does the same. Should we change the behavior to be consistent with those tools?

Reply:
I believe so. Additionally, we could emit a warning, and suggest passing the `-o` argument.

Reply:
Ok. If you're ok with the rest of the change, I'll just merge this as it is now and then follow up with the changes discussed above?